docs/Using Fleet/Audit-logs.md

```diff
@@ -686,6 +686,61 @@ This activity contains the following fields:
 }
 ```
 
+### Type `created_windows_profile`
```

Comment:
Note that the activity documentation (this file) is generated automatically from the code, so when I added the new activities it did update this documentation file (see https://github.com/fleetdm/fleet/pull/15045/files#diff-9f70e9133b8f91c2034329e45fbe2386fdc4fb0b27c114ce6e3b6f0310320551). It does seem identical at a glance.

Reply:
@mna I noted to delete these before merging. The idea was to have this PR as a reference (in terms of copy and names for activity types).